brave/brave-core
# Add installation progress to brave://ipfs page

Resolves https://github.com/brave/brave-browser/issues/13170

Added node installation progress to ipfs diagnostic page

![image](https://user-images.githubusercontent.com/2965009/110798833-9a8e1580-828b-11eb-8add-a402c6c171b2.png)

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Remove User Data
- Simulate slow network connection in sniffer
- open brave://ipfs and click install node
- you will see the installation progress
- if an error occurred you should be able to restart installation process
